application/controllers/ResponsesController.php

```diff
@@ -799,7 +798,7 @@ public function actionDeleteAttachments(int $surveyId, int $responseId = null):
             throw new CHttpException(405, gT("Invalid action"));
         }
 
-        $stringItems = json_decode($request->getPost('sItems'));
+        $stringItems = json_decode((string) $request->getPost('sItems'));
```

Comment:
Another place where you should use Yii functionality to assign default value instead of (string).